From 58ae2ba75fa57896bc690c57a892958eb9816cd4 Mon Sep 17 00:00:00 2001
From: Quentin Armitage <quentin@armitage.org.uk>
Date: Mon, 23 Apr 2018 17:05:15 +0100
Subject: [PATCH] Fix commit afc4508 Fix namespace collision with musl
 if_ether.h

Commit afc4508 used the wrong definitions for checking the if_ether.h
namespace collision. This commit uses the correct definitions.

Signed-off-by: Quentin Armitage <quentin@armitage.org.uk>
---
 keepalived/include/vrrp_arp.h  | 2 +-
 keepalived/include/vrrp_vmac.h | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/keepalived/include/vrrp_arp.h b/keepalived/include/vrrp_arp.h
index 3414d391..a206f502 100644
--- a/keepalived/include/vrrp_arp.h
+++ b/keepalived/include/vrrp_arp.h
@@ -25,7 +25,7 @@
 
 /* system includes */
 #include <netinet/in.h>
-#ifdef _HAVE_NET_LINUX_IF_H_COLLISION_
+#ifdef _HAVE_NETINET_LINUX_IF_ETHER_H_COLLISION_
 #define _NETINET_IF_ETHER_H
 #include <linux/if_ether.h>
 #endif
diff --git a/keepalived/include/vrrp_vmac.h b/keepalived/include/vrrp_vmac.h
index da14b240..6cba88fc 100644
--- a/keepalived/include/vrrp_vmac.h
+++ b/keepalived/include/vrrp_vmac.h
@@ -29,7 +29,7 @@
 #include <arpa/inet.h>
 #include <string.h>
 #include <syslog.h>
-#ifdef _HAVE_NET_LINUX_IF_H_COLLISION_
+#ifdef _HAVE_NETINET_LINUX_IF_ETHER_H_COLLISION_
 #define _NETINET_IF_ETHER_H
 #include <linux/if_ether.h>
 #endif
-- 
2.11.0

